src/index.html

```diff
@@ -17,11 +17,381 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
+    />
```

Comment:
This link to the stylesheet is redundant and can be removed. You've already linked to the stylesheet at line 20.